### PR TITLE
Updated Maki.gg due to website changes

### PR DESCRIPTION
**Pull request by m5rian**
My repository should be now up to date with the changes for the presence. Sorry for the inconvenience

### CODE REVIEW

**Review by Bas950 — CHANGES_REQUESTED** [review]
upload proof

**Comment by M5rian** [comment]
image

File: websites/M/Maki.gg/presence.ts [hunk 1]
```diff
@@ -136,5 +150,6 @@ presence.on("UpdateData", async () => {
   if (!presenceData.details) {
     presence.setTrayTitle();
     presence.setActivity();
-  } else presence.setActivity(presenceData);
+  } else
+    presence.setActivity(presenceData);
```
Comment (EncryptedDev) [inline websites/M/Maki.gg/presence.ts:154]: You are not required to have this check, alternatively, you can just do this: 
```suggestion
presence.setActivity(presenceData)
````

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/M/Maki.gg/presence.ts (hunk 1 above)
Comment (M5rian) [reply websites/M/Maki.gg/presence.ts]: Ah I see. Thank you

File: websites/M/Maki.gg/presence.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 const presence = new Presence({
-    clientId: "563434444321587202"
+  clientId: "563434444321587202"
```
Comment (EncryptedDev) [inline websites/M/Maki.gg/presence.ts:2]: Why did you remove the indent here?

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/M/Maki.gg/presence.ts (hunk 2 above)
Comment (M5rian) [reply websites/M/Maki.gg/presence.ts:2]: I think I understood it wrong, I though Codacy had issues with the spacing because it said "
Expected indentation of 4 spaces but found 2." Now the line starts with 4 spaces in front of it, is that wrong?

Comment (EncryptedDev) [reply websites/M/Maki.gg/presence.ts:2]: You have just removed the indentation for no reason lol

Comment (M5rian) [reply websites/M/Maki.gg/presence.ts:2]: Uhhh in my local repository I didn't remove them at all lol. Let me update my local one

**Review by EncryptedDev — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]
